docs/package.json

```diff
@@ -8,6 +8,9 @@
     "generate": "nuxi generate",
     "preview": "nuxi preview"
   },
+  "dependencies": {
+    "pnpm": "^7.27.0"
+  },
```

Comment:
The CI is failing because you added pnpm here as a dependency. Would you revert this change and the pagckage-lock.json change? That should do the trick.